### ISSUE TITLE
turning off running specific linters using yaml file

### ISSUE THREAD

**Issue by IndrajeetPatil** [jimhester/lintr#354]
I don't want specific `linters` to be run on my R package. How can I specify this in the `.lintr.yml` file?

This is what I currently have, but it doesn't seemt to be working.
```yaml
linters: with_defaults(snake_case_linter = NULL)
```

**Comment by russHyde** [comment]
The syntax looks OK to me and should prevent the snake_case linter from flagging any matching code. However, the linting settings are  usually stored in a file called `.lintr` rather than `.lintr.yml` - try again without the `.yml` extension.

Is `.lintr.yml` mentioned in the docs for `lintr` somewhere?

**Comment by IndrajeetPatil** [comment]
Sorry, my bad. I just assumed it was supposed to be a `.yml` file. 

That said, I have no idea how to create a Debian control file on a Windows machine. What's the file extension I am supposed to add when I save it?

![image](https://user-images.githubusercontent.com/11330453/48026397-f53f6180-e113-11e8-9b9d-0be7fc7e47e5.png)

**Comment by russHyde** [comment]
You don't have to select a file extension. Just leave 'save as type'  as 'All files' and put `.lintr` as the filename

**Comment by IndrajeetPatil** [comment]
Thanks!

**Comment by russHyde** [comment]
Are you using the github version or the CRAN version of lintr?

**Comment by IndrajeetPatil** [comment]
Awesome. That solved it!
(Now I am using the development version of `lintr`.)

**Comment by jimhester** [comment]
Thanks @russHyde!
